lib/react_client.dart

```diff
@@ -534,6 +565,7 @@ final ReactDartInteropStatics2 _dartInteropStatics2 = (() {
       });
 
   dynamic handleRender(Component2 component) => zone.run(() {
+        _updatePropsAndStateWithJs(component, component.jsThis.props, component.jsThis.state, component.jsThis.context);
```

Comment:
Can we pass in props/state/context as arguments directly here?

Going through `jsThis` is slower because it causes an interceptor lookup.